sdk/storage/test/blobs/performance_benchmark.cpp

```diff
@@ -36,4 +43,69 @@ namespace Azure { namespace Storage { namespace Test {
     }
   }
 
+  TEST_F(BlobContainerClientTest, DISABLED_MultiThreadPerf)
+  {
+    constexpr int concurrency = 64;
```

Comment:
This number should be tuned in the future. 64 is fine for this release.

Reply:
We can never find a number suitable for all kinds of hardware configuration. So you tune it for a specific machine, it still won't work for other machines.

Reply:
We can also specify the recommended machines here. It is difficult for maintainers to know how to run the perf test perfectly, but at least this would help achieve that goal.